packages/Frontend/src/components/LargeCard.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Card, Icon, Image } from 'semantic-ui-react';
+import SealedPortion from './assets/SealedPortion.jpg';
+
+
+export default class LargeCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      starFilled: false,
+    };
+  }
+  toggleFav = () => this.setState({ starFilled: !this.state.starFilled })
+  render() {
+    const { starFilled } = this.state;
+    return (
+      <div style={{ position: 'relative', display: 'inline' }}>
+        <Card>
+          <Icon
+            name="star"
+            color={starFilled ? 'yellow' : 'whitesmoke'}
+            onClick={this.toggleFav}
+            size="huge"
+            style={{
+              color: 'whitesmoke',
+              position: 'absolute',
+              zIndex: 1,
+              right: 0,
+              top: 9,
+            }}
+          />
+          <Image src={SealedPortion} />
+          <Card.Content>
+            <Card.Header>
+              The Sealed Portion
+            </Card.Header>
+          </Card.Content>
+          <Card.Content extra>
+            <p>
+              <Icon name="file text outline" />
+              22 hr read
```

Comment:
Change this "22 hr read" to be a prop that's passed in. The prop type will be a number, it'll be the number of seconds that we estimate it will take a reader to read the whole document from start to finish. Install moment.js and use their duration and humanize method to show the time in the proper format. https://momentjs.com/docs/#/durations/humanize/. Your code should be something like: `{moment.duration(this.props.readTime, 'seconds').humanize()}`. Moment will make it human readable, and they have great internationalization support, so when we add other languages it'll use them easily.